vendor/k8s.io/kubernetes/pkg/apis/componentconfig/types.go

```diff
@@ -473,6 +473,9 @@ type KubeletConfiguration struct {
 	// (binaries, etc.) to mount the volume are available on the underlying node. If the check is enabled
 	// and fails the mount operation fails.
 	ExperimentalCheckNodeCapabilitiesBeforeMount bool `json:"ExperimentalCheckNodeCapabilitiesBeforeMount,omitempty"`
+	// This flag, if set, instructs the kubelet to keep volumes from terminated pods mounted to the node.
+	// This can be useful for debugging volume related issues.
+	KeepTerminatedPodVolumes bool `json:"KeepTerminatedPodVolumes,omitempty"`
```

Comment:
Ok I see the others in this file do as well (upstream has since cleared them from internal). Nevermind on this for now.